.changeset/breezy-fans-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@voucherify/sdk': patch
+---
+
+Used FixMe convention for unknown types
```

Comment:
You don't need to create a changeset for every single change 😅 
If changes are to same package and all bump `patch`, you could just edit one of the changesets you've created in this PR:
```suggestion
---
'@voucherify/sdk': patch
---
Distributions types

- Used FixMe convention for unknown types
- Added Distributions types
- Removed deprecated method
- Updated Distribution types
```

It isn't an issue, you could leave it as is if you want (at the end all will be combined by GitHub action) but consider it for next time 😄